JavaScript/JavaScriptSDK/ajax/ajax.ts

```diff
@@ -159,6 +159,12 @@ module Microsoft.ApplicationInsights {
 
             if (CorrelationIdHelper.canIncludeCorrelationHeader(this.appInsights.config, xhr.ajaxData.getAbsoluteUrl())) {
                 xhr.setRequestHeader(RequestHeaders.requestIdHeader, xhr.ajaxData.id);
+                if (this.appInsights.context) {
+                    var appId = this.appInsights.context.appId();
```

Comment:
same comment as before. Just mention in the corresponding issue. This `appid` should be taken from ajaxData object's instrumentation key. This is why for instance we call telemetry initializers twice in .NET - making sure that ikey is populated on `DependencyTelemetry`